### DISCUSSION TITLE
Is there an option for part of the text within node to be one color and the rest of the text to be another color?

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1558] [Help]
I'd like for DOES NOT to be in red text, and the rest of the text to be in black. Would this be possible to accomplish? 
![Uploading dfsjdshufudssfd.png…]()

**Comment by i-plasm** [comment]
You have to enter the in-line core editor (by default, by double-clicking inside the node core). To format a portion of text just select the portion, then right-click so that the context menu will be displayed. In that menu choose the "Format" submenu and specify the color you want.

**Reply by adamfreeplaneuser01** [reply]
This is a great answer! I did not know about edit -> viewing. I still would like to see, however, an option to "lock" an individual node so that way people don't have to keep switching back and forth while working. If that's not already a feature, I hope that is something that could easily be implemented by the devs!

**Reply by i-plasm** [reply]
I think you meant to reply over the other thread :) . I'm not entirely sure I understand what you mean by locking an individual node. If it would be on a per-node basis, then wouldn't be it more cumbersome (and error-prone) to have to make each individual node (or a selection of them) editable/non-editable every time?

**Reply by adamfreeplaneuser01** [reply]
haha sorry, I am deep into my studying for my last final tonight. Basically all I'm doing right now is posting down ideas so that way we can discuss them further after my last final. This weekend, I'll elaborate more on what I'm thinking as far as this idea goes!

**Reply by adamfreeplaneuser01** [reply]
[i-plasm](https://github.com/i-plasm)
[bbarbosa](https://github.com/i-plasm)

I'm not sure how to tag another person on Github, so hopefully you see this lol

https://github.com/freeplane/freeplane/assets/147225942/8c06eb79-a4be-4553-97a3-8b2d75e41e56

**Reply by i-plasm** [reply]
Hi, hope the exams went well! When you double click the node core, the editor gets activated - which you called something like "I get an option to edit the text" => exactly, that's the in-line core editor. From there you can change the font color of portions of text, check this video:
![core-editor-font](https://github.com/freeplane/freeplane/assets/144830522/4df87880-a268-411b-81a8-60778bab656e)
